README.md

```diff
@@ -72,9 +83,16 @@ Install the following for best results:
 
 ## Using the application with Internet Explorer 11
 
-If using Internet Explorer 11 with Windows 7, download the Adobe Reader application at
-https://get.adobe.com/reader/
-Install as directed. This will permit PDF documents to be viewed in-browser with Internet Explorer 11.
+If using Internet Explorer 11 with Windows 7, download the Adobe Reader application at https://get.adobe.com/reader/. Install as directed. This will permit PDFs to be viewed in-browser with Internet Explorer 11.
+
+## Forked dependencies
+The software has several dependencies that required minor modifications to suit our needs. Rather than attempt to persuade their creators to adopt our modifications, those repositories have been forked within the U.S. Tax Court's GitHub organization, and the modifications made there. Those repsitories are:
```

Comment:
tiny typo in the second instance of the word 'repositories' in this para

Reply:
Ha! I mistyped it, "fixed" it, and...apparently got it wrong the second time, too. :)